modules/govuk_jenkins/templates/jobs/transition_import_dns.yaml.erb

```diff
@@ -0,0 +1,23 @@
+---
+- job:
+    name: transition_import_dns
+    display-name: Transition: daily DNS import
+    project-type: freestyle
+    description: "<p>Performs a daily update on the DNS records for each transitioned domain name</p>"
+    logrotate:
+      numToKeep: 30
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          artifact-num-to-keep: 5
+    builders:
+        - shell: |
+            HOSTNAME=$(govuk_node_list -c backend --single-node)
+            ssh deploy@$HOSTNAME "cd /var/apps/transition && govuk_setenv transition bundle exec rake import:dns_details"
```

Comment:
I've looked at the code for this in transition, and I don't think this task updates the CDN:

- https://github.com/alphagov/transition/blob/master/lib/tasks/import/dns_details.rake
- https://github.com/alphagov/transition/blob/master/lib/transition/import/dns_details.rb

Am I missing something?

Reply:
This doesn't update the CDN - it's the https://github.com/alphagov/govuk-puppet/pull/9319/files#diff-b19845cf2511fa96f3d5d029b7f20742R32 task.

The previous task: https://github.com/alphagov/govuk-puppet/pull/9319/files#diff-e82a80955f654ee3bc23fb314b8320e2L24 did [3 things](https://github.com/alphagov/transition/blob/3f0feab8599a2a2e6dfb7004e8506aaad7655e31/lib/tasks/import/all.rake#L7-L11) which included the CDN update.

This one is just moved since there doesn't seem to be a particular reason to run it in the middle of night so why bother?

Reply:
Right, I've just worked my way through the jobs.  The old job triggered the bouncer CDN job:

```yaml
    publishers:
        - trigger-parameterized-builds:
            - project: Bouncer_CDN
              condition: 'SUCCESS'
              trigger-with-no-params: true
```

I think this job should trigger it too.

Reply:
@barrucadu I don't think the DNS job needs to trigger the CDN job as this isn't included in any data sent to Fastly. 

I'm not really sure what the DNS information is used for - do we proxy some sites? But it is displayed on admin pages: https://transition.integration.publishing.service.gov.uk/sites/defra

Reply:
Maybe the CDN job doesn't need to be triggered by this job, but what stands out to me is that the CDN job was triggered by the old job, and now none of the new jobs trigger it.  I'm not really sure what the DNS information is used for either.

Reply:
The CDN job needs to run when the `import:all:orgs_sites_hosts` job runs. This is still the case for the job that runs it: https://github.com/alphagov/govuk-puppet/blob/9298766ca7109f6ad81f1a5dec51afc7d30a874a/modules/govuk_jenkins/templates/jobs/transition_load_site_config.yaml.erb#L40-L44

The job that has been split into two used to do three things: import DNS, import hits and import orgs sites hosts. The last of these has been dropped and that was the only one which needed the CDN deploy.